daemon.c

```diff
@@ -112,19 +112,46 @@ void logmsg(int prio, const char *fmt, ...)
 }
 
 /*
- * return remote address from svc_req structure
+ * return remote address from svc_req structure in IPv6 format
+ * 
+ * IPv4 addresses are mapped to IPv6
+ *
+ * for AF_INET6, remote_buf is not used and ptr inside svc_req struct is returned
+ * for AF_INET, remote_buf is used and its poineter is returned
```

Comment:
Pokud je komentář víc, než jedna věta, tak věty začínejte velkým písmenem a na konci dělejte tečku. A napište to jako:

For AF_INET6 addresses, remote_buf is not used and a pointer to a field of svc_req is returned.
For AF_INET addresses, the address is stored into the remote_buf and its address is returned.

Reply:
Zde bych to změnil na:

For AF_INET6 addresses, remote_buf is not used and a pointer to a field of netbuf (provided by svc_req) is returned.

Reply:
OK